__tests__/index.test.ts

```diff
@@ -184,7 +193,7 @@ describe('#validate', () => {
     ['OpenAPI 3.1', '3.1'],
   ])('%s support', (_, version) => {
     it('should validate a URL hosting JSON as expected', async () => {
-      const json = require(`@readme/oas-examples/${version}/json/petstore.json`);
+      const json = await import(`@readme/oas-examples/${version}/json/petstore.json`).then(r => r.default);
```

Comment:
god I love discovering how comprehensive your tests are in all these random libraries lol